src/Glpi/Form/Migration/FormMigration.php

```diff
@@ -592,6 +594,59 @@ private function updateBlockHorizontalRank(): void
         }
     }
 
+    /**
+     * The new form system no longer allows defining a precise width for a block
+     * The width of each block is now determined by the number of elements contained in the horizontal layout
+     */
+    private function updateHorizontalRanks(): void
+    {
+        $this->progress_indicator?->setProgressBarMessage(__('Updating horizontal ranks...'));
```

Comment:
Same here, I am not sure "horizontal rank" is something that users will understand easily.

Reply:
`Fixing forms layouts...`